README.md

```diff
@@ -10,26 +10,39 @@ The documentation is built using Markdown pages, which can be converted into a n
 
 ## Build dependencies
 
-The documentation site uses [Hugo](https://gohugo.io/), and specifically uses features found in the extended version of Hugo, v0.76.0 and later. See `.tool-versions` for the currently used version
+The documentation site uses [Hugo](https://gohugo.io/) including the extended version of Hugo. Version v0.122 is used per (see #https://github.com/medic/cht-docs/issues/1302) as denoted in `.tool-versions` [file](https://github.com/medic/cht-docs/blob/main/.tool-versions).
 
-[asdf](https://asdf-vm.com/guide/getting-started.html) is the recommended way to manage `hugo` and `golang` versions for local development.
+To proceed, please choose one of the three options below of Docker, native packages or using `asdf` then proceed to ["Run Hugo" below](#run-hugo) to start your local instance.
 
-After installing it run:
+### Docker
+
+Running `hugo` locally using Docker is likely the easiest way to update the docs.  You only need to ensure you have [Docker and Docker compose installed](https://docs.docker.com/compose/install/).
+
+### Native packages
+
+If you'd like to install packages directly on your workstation, follow the [installation instructions for your Operating System](https://gohugo.io/getting-started/installing/), and be sure to get the extended version. Most users will be able to simply install using their native package manager like `brew`, `apt` or `snap`.
```

Comment:
Probably should also warn folks here that they need to be aware of the Hugo version they are installing (and if it matches the `.tool-versions`.

Reply:
yup yup!  Fixed in 685431f6